### ISSUE TITLE
Get holiday name of a specific date

### ISSUE THREAD

**Issue by thetric** [sfakir/feiertagejs#8]
There already exists `getHolidays` that returns an array with all holidays of a year. Can you please add a method to get the holiday name of _one_ date?

**Comment by sfakir** [comment]
Hello thetric,
thank you for the request. What would be the expected input of this method - 
year + constant ( as following?) 

let holiday =  feiertagejs.getHoliday('2016','NEUJAHRSTAG');

holiday.date // = Date("2016-01-01");
holiday.name // 'NEUJAHRSTAG' (constant)
holiday.trans() // German translation: Neujahrstag
holiday.equals(date) // Compare days only (ignore time)
--
alternativley it could be  the date
let holiday =  feiertagejs.getHoliday(new Date(2016,1,1));  ?

**Comment by thetric** [comment]
Thank you for your quick reply!
I would expect something like your last proposal:

```js
const holiday = feiertragejs.getHoliday(new Date(2016, 0, 1), 'BUND');
```

For my use case it would be enough to simply return the holiday's name (e.g. "Neujahr"), but to be consistent with `getHolidays()` the new method should return an object like in your first snippet.

**Comment by sfakir** [comment]
Hi there,
sounds reasonable to me. Please check the new method '**getHolidayByDate**' published in v1.1.7 and let me know if it fulfills your needs.

**Comment by thetric** [comment]
Sure, I'll check it tomorrow. Thank you for fast reaction :)
